src/blocked.c

```diff
@@ -65,23 +65,37 @@
 #include "latency.h"
 #include "monotonic.h"
 
-void serveClientBlockedOnList(client *receiver, robj *o, robj *key, robj *dstkey, redisDb *db, int wherefrom, int whereto, int *deleted);
-int getListPositionFromObjectOrReply(client *c, robj *arg, int *position);
-
-/* This structure represents the blocked key information that we store
- * in the client structure. Each client blocked on keys, has a
- * client->bpop.keys hash table. The keys of the hash table are Redis
- * keys pointers to 'robj' structures. The value is this structure.
- * The structure has two goals: firstly we store the list node that this
- * client uses to be listed in the database "blocked clients for this key"
- * list, so we can later unblock in O(1) without a list scan.
- * Secondly for certain blocking types, we have additional info. Right now
- * the only use for additional info we have is when clients are blocked
- * on streams, as we have to remember the ID it blocked for. */
-typedef struct bkinfo {
-    listNode *listnode;     /* List node for db->blocking_keys[key] list. */
-    streamID stream_id;     /* Stream ID if we blocked in a stream. */
-} bkinfo;
```

Comment:
maybe we don't have to move this big block?
it's was never accessed anywhere outside blocked.c even before,
but there's a server struct variable holding this struct and it still has a comment referring to it.

Reply:
I am afraid I do not understand. the bkinfo is gone (for now) do you mean the comment itself?

Reply:
sorry, either i commented on the wrong block, or somehow GH quoted the wrong thing.
i meant for the `readyList` struct and its comment.

Reply:
done